lib/explorer/data_frame.ex

```diff
@@ -3150,9 +3150,8 @@ defmodule Explorer.DataFrame do
   @doc """
   Arranges/sorts rows by columns using a callback function.
 
-  The callback receives a lazy dataframe. A lazy dataframe does
-  hold any values, instead it stores all operations in order to
-  execute all sorting performantly.
+  The callback receives a lazy dataframe. A lazy dataframe does not
+  hold values; instead it stores operations for efficient sorting.
```

Comment:
TIL!

I think using `; instead` increases the barrier for contribution though... as we are going into advanced english. Could we use something else? "rather"? remove "instead" altogether? Thoughts?

Reply:
If both ";" and "," trip folks up for one reason or another, here's an alternative:

> The callback receives a lazy dataframe which stores operations instead of values for efficient sorting.

And this is off topic, but we (I) might want to consider writing a guide on lazy frames. It's a somewhat complex topic. If you come from Pandas there's no equivalent AFAIK.

Reply:
> TIL!
> 
> I think using `; instead` increases the barrier for contribution though... as we are going into advanced english. Could we use something else? "rather"? remove "instead" altogether? Thoughts?

I think the semicolon is correct, but I normally use a dash or emdash....they seem to make more sense to me.

> A lazy dataframe does not hold values–instead it stores operations for efficient sorting.

But I like Billy's suggestion.